src/components/Template/Template.test.tsx

```diff
@@ -1,330 +1,156 @@
-const header = <Placeholder variant="short">NYPL Header</Placeholder>;
-const breakout = (
-  <>
-    <Placeholder variant="short">Breadcrumbs</Placeholder>
-    <Placeholder variant="short">Hero</Placeholder>
-  </>
-);
-const sidebar = "left";
+const breakout = <Placeholder variant="short">Breakout</Placeholder>;
 const contentTop = <Placeholder>Content Top</Placeholder>;
-const contentSidebar = <Placeholder>Left Sidebar</Placeholder>;
-const contentPrimary = (
+const contentMain = (
   <>
     <Placeholder>Main Content</Placeholder>
     <Placeholder variant="short">More Content</Placeholder>
   </>
 );
-const footer = <Placeholder variant="short">Footer</Placeholder>;
+const contentBottom = <Placeholder variant="short">Content Bottom</Placeholder>;
+
+const templateComponents = (
+  sidebar: SidebarPlacement = "none",
+  useMainNarrow: boolean = false
+) => (
+  <Template sidebar={sidebar}>
+    <TemplateBreakout>{breakout}</TemplateBreakout>
+    <TemplateTop>{contentTop}</TemplateTop>
+    {sidebar !== "none" && !useMainNarrow && (
+      <TemplateSidebar>
+        <Placeholder>{sidebarLabel(sidebar)}</Placeholder>
+      </TemplateSidebar>
+    )}
+    {useMainNarrow ? (
+      <TemplateMainNarrow>{contentMain}</TemplateMainNarrow>
+    ) : (
+      <TemplateMain>{contentMain}</TemplateMain>
+    )}
+    <TemplateBottom>{contentBottom}</TemplateBottom>
+  </Template>
+);
 
-describe("TemplateAppContainer accessibility", () => {
+describe("Template components accessibility", () => {
   it("passes axe accessibility test", async () => {
-    const { container } = render(
-      <TemplateAppContainer
-        aboveHeader={aboveHeader}
-        header={header}
-        breakout={breakout}
-        sidebar={sidebar}
-        contentTop={contentTop}
-        contentSidebar={contentSidebar}
-        contentPrimary={contentPrimary}
-        footer={footer}
-      />
-    );
+    const { container } = render(templateComponents());
     expect(await axe(container)).toHaveNoViolations();
   });
-});
 
-describe("Template components accessibility", () => {
   it("passes axe accessibility test", async () => {
-    const { container } = render(
-      <Template>
-        <TemplateAboveHeader>{aboveHeader}</TemplateAboveHeader>
-        <TemplateHeader>
-          {header}
-          <TemplateBreakout>{breakout}</TemplateBreakout>
-        </TemplateHeader>
-        <TemplateContent sidebar={sidebar}>
-          <TemplateContentTop>{contentTop}</TemplateContentTop>
-          <TemplateContentSidebar>{contentSidebar}</TemplateContentSidebar>
-          <TemplateContentPrimary>{contentPrimary}</TemplateContentPrimary>
-        </TemplateContent>
-        <TemplateFooter>{footer}</TemplateFooter>
-      </Template>
-    );
+    const { container } = render(templateComponents("left"));
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("passes axe accessibility test", async () => {
```

Comment:
These four tests have the same description. Update this so they are all descriptive to the test.